test/ReactTouchHandler-test.js

```diff
@@ -1,46 +1,34 @@
 /**
  * Copyright Schrodinger, LLC
  */
-
-import React from 'react';
 import ReactTouchHandler from '../src/ReactTouchHandler';
 import sinon from 'sinon';
 import { assert } from 'chai';
-import { createRenderer, isElement } from 'react-addons-test-utils';
 
 describe('ReactTouchHandler', function() {
-  var clock, sandbox, requestAnimationFramePolyfillSpy;
-
-  before(function () {
-    clock = sinon.useFakeTimers();
```

Comment:
since all tests share a sandbox I moved this down to the test which actually uses it.

Reply:
nit: clock is defined below in the actual test, so we don't need the declaration here.

Reply:
good catch, will do